docs/content/token/staking2/stakers.mdx

```diff
@@ -32,45 +32,105 @@ _Note: The corresponding **node operator** steps are outlined in a
 
 ## 1. Fetch node operator information
 
-TODO
-- Provide `operatorAddress, nodeID`
-- Uses `TH.04` script
+To fetch the node operator information for an address's staking proxy, the token holder must use use script TH.04:
+
+`stakingProxy/sh_get_node_info.cdc`
+
+Script arguments:
+
+| Argument     | Type    | Purpose |
+|--------------|---------|---------|
+| **account**  | Address | Specifies the address to get the staking proxy's node info node info for. |
+
+Script returns:
+
+The `StakingProxy.NodeInfo` struct containing the details for the node. If no NodeInfo is available, the script will fail.
 
 ## 2. Register node & grant staking access to the node operator
 
-TODO
-- Provide `amount` to be staked
-- Provide `operatorAddress, nodeID` to stake to
-- Uses `TH.05` transaction
+To register the node as the token holder account's staker, the token holder must use  use transaction TH.05:
```

Comment:
Can you make it so all of the transaction IDs (e.g. `TH.05`) link to the relevant section of `transactions.mdx`? I know you can't link directly to the line in the table, but linking to the whole table is enough I think

Reply:
I've done this in rob/docs/staking-as-token-holder .

Reply:
Done.